.github/workflows/vib-build.yml

```diff
@@ -79,6 +79,8 @@ jobs:
 
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v3
+      with:
+        buildkitd-flags: --allow-insecure-entitlement security.insecure
```

Comment:
For what reason is this needed?

Reply:
This is what we can do to grant privilege in the workflow. Ref: https://github.com/docker/build-push-action/issues/717